packages/rum-core/src/domain/view/trackViews.ts

```diff
@@ -281,6 +313,7 @@ function newView(
     service,
     version,
     context,
```

Comment:
**💬 suggestion:** ‏Remove this `context` variable and property, and use `currentView.contextManager.getContext()` above (single source of truth, also right now `context` does not reflect the latest context of the view)